pkg/lint/rules/chartfile_test.go

```diff
@@ -65,9 +67,15 @@ func TestValidateChartYamlFormat(t *testing.T) {
 }
 
 func TestValidateChartName(t *testing.T) {
-	err := validateChartName(badChart)
-	if err == nil {
-		t.Errorf("validateChartName to return a linter error, got no error")
+	badNameChart, _ := chartutil.LoadChartfile(goodChartFilePath)
```

Comment:
looks to be a mismatch here: `badNameChart := f(goodChartFilePath)`

Reply:
Thank you for the comment. In the next lines I change the name of the good chart to one with a "bad" name. I used the good chart as a reference so that the rest is correct, only the name is "bad". Hope this makes sense. 

The test catches the bad names so is working as it should.